app/src/main/java/com/example/deejai/RoomSelection/RoomAdapter.kt

```diff
@@ -1,5 +1,8 @@
 package com.example.deejai.RoomSelection
 
+import android.content.Context
+import android.content.Intent
+import android.util.Log
```

Comment:
If this was a debugging log, please remove it.

Reply:
done